src/TConsole.cpp

```diff
@@ -369,15 +372,15 @@ void TConsole::Command_Kick(const std::string&, const std::vector<std::string>&
     bool Kicked = false;
     // TODO: this sucks, tolower is locale-dependent.
     auto NameCompare = [](std::string Name1, std::string Name2) -> bool {
```

Comment:
More a style thing than anything else, but I'd prefer never passing them as value unless we need to ;)
```suggestion
    auto NameCompare = [](const std::string& Name1, const std::string& Name2) -> bool {
```